config.default.yaml

```diff
@@ -300,19 +320,19 @@ renewable:
     resource:
       method: hydro
       hydrobasins: data/hydrobasins/hybas_world.shp
-      flowspeed: 1.0  # m/s
+      flowspeed: 1.0 # m/s
       # weight_with_height: false
       # show_progress: true
     carriers: [ror, PHS, hydro]
     PHS_max_hours: 6
-    hydro_max_hours: "energy_capacity_totals_by_country"  # not active
-    hydro_max_hours_default: 6.0  # (optional, default 6) Default value of max_hours for hydro when NaN values are found
+    hydro_max_hours: "energy_capacity_totals_by_country" # not active
+    hydro_max_hours_default: 6.0 # (optional, default 6) Default value of max_hours for hydro when NaN values are found
     clip_min_inflow: 1.0
-    extendable: true
+    extendable: false
```

Comment:
damn, I should have written it down, there were some scenarios where the hydro was expanded. in pypsa-eur, we don't have hydro as extendable, since they are most expanded to their feasible limit. what is your experience with the expansion of hydro?

Reply:
reverted

Reply:
thanks for sharing your experience with expansion of hydro!
in general, absolutely agree that it may be quite delicate point which should be tracked and improved